packages/hatchify-core/src/types/schema.ts

```diff
@@ -1,4 +1,7 @@
-export type Attribute = string | { type: string }
+export type Attribute =
+  | string
+  | { type: string }
```

Comment:
if you're talking about `{ type: string }` then no - this is here because in v2 schema we can have more verbose attribute definitions, for example:
```
age: "number",
birthYear: { type: "number", ...some validation stuff (min, max, custom) },
```
though with the current setup it looks like you _could_ technically do `type: "enum"` and then not set values. once v2 schema is finalized this bit would need to be cleaned up so that `type` when in an object without values can only be `date/string/boolean/number/+whatever else i am missing` :D

Reply:
the typescript checker won't stop you but Sequelize will throw `Error: Values for ENUM have not been defined.` if you configure the ENUM type with no values or empty values.